src/app/screens/Accounts/index.tsx

```diff
@@ -195,6 +242,72 @@ function AccountsScreen() {
             </div>
           </form>
         </Modal>
+
+        <Modal
+          closeTimeoutMS={200}
+          isOpen={exportModalIsOpen}
+          onRequestClose={closeExportModal}
+          contentLabel="Edit account name"
+          overlayClassName="bg-black bg-opacity-25 fixed inset-0 flex justify-center items-center p-5"
+          className="rounded-lg bg-white w-full max-w-lg"
+        >
+          <div className="p-5 flex justify-between dark:bg-surface-02dp">
+            <h2 className="text-2xl font-bold dark:text-white">
+              Export account
+            </h2>
+            <button onClick={closeExportModal}>
+              <CrossIcon className="w-6 h-6 dark:text-white" />
+            </button>
+          </div>
+
+          {loading && (
+            <div className="p-5 flex justify-center items-center space-x-2 dark:text-white">
+              <Loading />
+              <span>waiting for LndHub data...</span>
+            </div>
+          )}
+          {!loading && (
+            <div className="p-5 border-t border-b border-gray-200 dark:bg-surface-02dp dark:border-neutral-500">
+              {lndHubData.lnAddress && (
+                <div className="dark:text-white mb-6">
+                  <p>
+                    <strong>Your lightning address:</strong>
```

Comment:
```suggestion
                    <strong>Your Lightning Address:</strong>
```

I think we write it like this everywhere else